### DISCUSSION TITLE
Slide deck about the SCI specification

### DISCUSSION THREAD

**Discussion by iivanoo** [Green-Software-Foundation/discussions#168] [Ask Anything]
Hello!

I am an Associate Professor at VU Amsterdam and, together with [Vincenzo Stoico](https://github.com/vnzstc), I am coordinating a master course called Green Lab. 
The course is about empirical methods applied to energy-efficient software. [Here](https://arxiv.org/abs/2407.05689) you can have an idea about the structure and contents of the course.

I am writing since we are planning to teach the SCI spec to our students and to ask them to include it in their project.
Is there by any chance any slide deck about the SCI we might reuse during the course?

Thanks in advance!

  Ivano

**Comment by NAMRATA-WOKE** [comment]
@iivanoo Thank you for informing us, and we're so excited to hear the SCI Specification will be taught at VU Amsterdam. 

I'm sharing some key messages and an image that you can use:

- Software Carbon Intensity (SCI) Specification: A specification and standardized protocol to calculate a carbon intensity score for software applications.
- Measures the rate of carbon emissions for any type of application. 
- Enables users and developers to make informed choices regarding tools, architectures, and services by incorporating carbon usage considerations.
- Does not incorporate neutralizations or offsets into its calculations.
- It is a score rather than a total; lower numbers are better than higher numbers, and reaching 0 is impossible.
- The latest version 1.1 of the SCI Specification is now available [here](https://sci.greensoftware.foundation/).

![GSF Q1 report](https://github.com/user-attachments/assets/8cdfe160-3f48-4d78-9ac3-1c0870a5f281)

@seanmcilroy29 Is there a presentation deck that we have on the SCI Spec that can be passed along to Ivano?

**Reply by iivanoo** [reply]
Thanks @NAMRATA-WOKE for the material!

Let's see if @seanmcilroy29 has a slide deck for us, otherwise I will start preparing one (and maybe share it to you?).

**Reply by iivanoo** [reply]
Hi @seanmcilroy29, did you have time to check if you have a slide deck about the SCI spec? Thanks!

**Comment by iivanoo** [comment]
Hi all, I eventually did the slide deck myself, here it is: [https://www.ivanomalavolta.com/the-software-carbon-intensity-sci/](https://www.ivanomalavolta.com/the-software-carbon-intensity-sci/)

Feel free to reuse the slides as you see fit!
